ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.js

```diff
@@ -250,44 +257,40 @@ export default {
       run: (data) => delete data.hatch,
     },
     {
-      id: 'UCU Twintania P2',
-      regex: Regexes.hasHP({ name: 'Twintania', hp: '75', capture: false }),
-      regexDe: Regexes.hasHP({ name: 'Twintania', hp: '75', capture: false }),
-      regexFr: Regexes.hasHP({ name: 'Gémellia', hp: '75', capture: false }),
-      regexJa: Regexes.hasHP({ name: 'ツインタニア', hp: '75', capture: false }),
-      regexCn: Regexes.hasHP({ name: '双塔尼亚', hp: '75', capture: false }),
-      regexKo: Regexes.hasHP({ name: '트윈타니아', hp: '75', capture: false }),
+      id: 'UCU Twintania Phase Change Watcher',
+      type: 'Ability',
+      // On Twister or Generate.
+      netRegex: NetRegexes.startsUsing({ id: '26A[AE]', source: 'Twintania' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '26A[AE]', source: 'Twintania' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '26A[AE]', source: 'Gémellia' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '26A[AE]', source: 'ツインタニア' }),
+      netRegexCn: NetRegexes.startsUsing({ id: '26A[AE]', source: '双塔尼亚' }),
+      netRegexKo: NetRegexes.startsUsing({ id: '26A[AE]', source: '트윈타니아' }),
+      condition: (data) => !data.monitoringHP && data.hpThresholds[data.currentPhase] !== undefined,
+      preRun: (data) => data.monitoringHP = true,
+      promise: (data, matches) => Util.watchCombatant({
+        ids: [parseInt(matches.sourceId, 16)],
+      },
+      (ret) => {
+        return ret.combatants.some((c) => {
+          const currentHPCheck = data.hpThresholds[data.currentPhase] ?? -1;
+          return currentHPCheck >= (c.CurrentHP / c.MaxHP);
```

Comment:
I know these are my doing, but to nit:
```suggestion
          return c.CurrentHP / c.MaxHP <= currentHPCheck ;
```
Just to match the new ones you added (I feel this form is more readable: "return true if any combatants have a lower HP % than the current HP check").

Reply:
Done, thanks!